tests/basic_correctness/test_basic_correctness.py

```diff
@@ -130,7 +130,7 @@ def test_models_distributed(
     # Import VLLM_USE_V1 dynamically to handle patching
     from vllm.envs import VLLM_USE_V1
     if VLLM_USE_V1 and distributed_executor_backend != "mp":
-        pytest.skip(f"Skip {distributed_executor_backend} for V1")
+        os.environ["VLLM_ENABLE_V1_MULTIPROCESSING"] = "0"
```

Comment:
It does not work with ray executor. But I think it makes sense this option is for MP only as Ray is a different executor.

Reply:
I don't think this should have anything to do with the executor that's used. https://github.com/vllm-project/vllm/pull/9826 is the PR that introduced it and has a nice diagram of what's going on.

What happens if we try to run the Ray executor with the AsyncLLM?

Reply:
When using `VLLM_ENABLE_V1_MULTIPROCESSING`, it actually caused ray worker initialization to hang, because of an uninitialized ray environment in the new process.

Reply:
We should fix this, since AsyncLLM allows detokenization to be overlapped with the forward pass, and is the default in V1 now.

We probably want to fix this by initializing the ray environment in the entry point for the process that owns the executor:
https://github.com/vllm-project/vllm/blob/48edab8041741a82a1fd2f4d463cc0f393561b05/vllm/v1/engine/core.py#L239-L276

I don't think this should be a blocker for this PR but could you look into fixing this in a follow-up soon?

Reply:
Thanks, sounds good. I will follow up soon.